tools/importer/rules/notes.js

```diff
@@ -0,0 +1,63 @@
+
+export default function transformNotes(main, document) {
+  const helpxNotes = document.querySelectorAll('.helpx-note, .helpx-caution');
+  helpxNotes.forEach((el) => {handleHelpxNote(el, document);});
+
+  const alertContainers = document.querySelectorAll('.alert-container');
+  alertContainers.forEach((el) => {handleAlertContainer(el, document);});
+}
+
+
+
+function handleHelpxNote(el, document) {
+  const note = {};
+
+  let style = 'note';
+  if (el.classList.contains('helpx-caution')) {
+    style = 'caution';
+  }
+
+  // find the title text
+  const title = el.querySelector('.note-title');
+  if (title) {
+    note.title = title.innerHTML.replace(/[\n\t]/gm, '');
+  }
+
+  // find the text
+  const text = el.querySelector('.cmp-text');
+  if (text) {
+    note.text = text.innerHTML.replace(/[\n\t]/gm, '');
+  }
+  
+  const cells = [
+    ['Note'],
+    [note.text],
+    ['type', style],
+  ];
+  const table = WebImporter.DOMUtils.createTable(cells, document);
+
+  el.insertAdjacentElement('beforebegin', table);
+}
+
+
+
+function handleAlertContainer(el, document) {
+  console.log("handleAlertContainer", el);
```

Comment:
can we remove the console logs and commented code if not needed?

Reply:
Done in [9d8a6d2](https://github.com/adobecom/helpx-internal/pull/1/commits/9d8a6d25e7dfbc7248e81ef5227d88d278dd5f8b)